src/helm/clients/openai_client.py

```diff
@@ -55,7 +55,12 @@ def _is_chat_model_engine(self, model_engine: str) -> bool:
             return True
         return False
 
+    def _is_high_res_vision_model(self, model_engine: str) -> bool:
+        return model_engine == "gpt-4-vision-preview-high-res"
+
     def _get_model_for_request(self, request: Request) -> str:
+        if self._is_high_res_vision_model(request.model_engine):
+            return "gpt-4-vision-preview"
```

Comment:
Future-proof this: `return request.model_engine.replace("-high-res", "")`

Otherwise there will be a bug in the future if someone adds another res vision model but forgets to change this.

Reply:
Actually, I removed the high-res version. It doesn't really do anything for small images and I think we should add a parameter in AdpaterSpec if we more APIs or models support it down the road.